packages/cli/src/oclif/ZapierBaseCommand.js

```diff
@@ -0,0 +1,52 @@
+const { Command } = require('@oclif/command');
+
+const { startSpinner, endSpinner } = require('../utils/display');
+
+const inquirer = require('inquirer');
+
+class ZapierBaseCommand extends Command {
+  run() {
+    this.parseFlags();
+    return this.perform();
+  }
+
+  parseFlags() {
+    // normally this is called via `this.parse(SomeCommand)`, but that's error-prone and I got tired of typing it
+    // .constructor is the static class
+    const { flags, args } = this.parse(Object.getPrototypeOf(this).constructor);
+
+    this.flags = flags;
+    this.args = args;
+  }
+
+  perform() {
+    throw new Error('subclass me');
+  }
+
+  // UTILS
+  log(...message) {
+    if (!['json', 'raw'].includes(this.flags.format)) {
+      super.log(...message);
+    }
+  }
+
+  async confirm(message, defaultAns = false) {
+    const { ans } = await inquirer.prompt({
+      type: 'confirm',
+      message,
+      default: defaultAns,
+      name: 'ans'
+    });
+    return ans;
+  }
+
+  startSpinner(message) {
+    startSpinner(message);
+  }
+
+  stopSpinner({ success = true, message = undefined } = {}) {
+    endSpinner(success, message);
+  }
```

Comment:
We're wrapping `startSpinner` and `endSpinner` into `ZapierBaseCommand` because it saves us an import in the subclass, correct? If so, I don't think it brings too much convenience. I'd suggest removing the two.

Reply:
The main reasoning is that eventually I'd like all presentational logic to be on the base command and all business logic in util files. 

For now, it makes more sense to leave the spinner code where it is (simpler for existing commands), but eventually we'll be able to move it onto the base command. When that happens, all uses of it will already be using `this.startSpinner()`, so no extra action will be needed.

Reply:
That makes sense. Thanks for the clarification!